src/components/AnnotationCreator.jsx

```diff
@@ -249,6 +230,15 @@ export default function AnnotationCreator({
           id="SAVE"
         />
       </DialogActions>
+      {error && (
+        <CustomAlert
+          titleId="SERVER_ERROR"
+          style={{ marginTop: 16, marginBottom: 8 }}
+          severity="error"
+        >
+          {error}
+        </CustomAlert>
+      )}
```

Comment:
does it look weird for the error to be below the buttons? everywhere else we do the errors above them

Reply:
Ok... moved it above. It looks better, I agree:
<img width="1552" alt="Screen Shot 2022-04-07 at 5 29 34 PM" src="https://user-images.githubusercontent.com/2775448/162340289-e2fdf3b4-77b4-4377-8f55-2185a5ab05cf.png">